doc/CMakeLists.txt

```
@@ -19,20 +19,20 @@ find_program(SPHINX_EXECUTABLE
         DOC "Sphinx documentation generation tool (http://www.sphinx-doc.org/)"
         )
 
-if ( SPHINX_EXECUTABLE )
+if (SPHINX_EXECUTABLE)
     set(SPHINX_SOURCE ${CMAKE_CURRENT_SOURCE_DIR}/sphinx)
     set(SPHINX_BUILD ${CMAKE_CURRENT_BINARY_DIR}/sphinx)
 
     add_custom_target(sphinx
-            ${SPHINX_EXECUTABLE} -b html ${SPHINX_SOURCE} ${SPHINX_BUILD}/html
-            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+            COMMAND ${CMAKE_COMMAND} -E env PYTHONPATH=${CMAKE_BINARY_DIR}:$ENV{PYTHONPATH} ${SPHINX_EXECUTABLE} -b html ${SPHINX_SOURCE} ${SPHINX_BUILD}/html
```

```suggestion
            COMMAND ${CMAKE_COMMAND} -E env PYTHONPATH=${TEST_ENV} ${SPHINX_EXECUTABLE} -b html ${SPHINX_SOURCE} ${SPHINX_BUILD}/html
```
And I will replace `TEST_ENV` later everywhere.

---

ok